client/src/components/new-request/analyze-results.vue

```diff
@@ -135,9 +135,7 @@
                 <v-col cols="12" class="copy-normal pt-2 pb-4">
                   <v-row justify="center">
                     <v-col cols="auto">
-                      Name is available for a
-                      {{ entityText === 'BC Corporation' && location.text === 'BC' ? '' : location.text }}
-                      {{ ' ' + entityText }}
+                      Your name looks good to send to BC Registries Names Staff for a final decision.
```

Comment:
The design in the ticket says "This name...".

Did this get changed? If not, please update. Thx.

Reply:
(Personally I prefer "Your name..." but it isn't up to me.)